src/CommunityToolkit.Maui/Views/Snackbar/SnackBar.shared.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Threading.Tasks;
+using CommunityToolkit.Maui.Views;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+
+
+namespace CommunityToolkit.Maui.Controls.Snackbar;
+
+public class Snackbar : BasePopupView, IText
```

Comment:
Since we're building a library, I think we should implement our views on top of interfaces, like .net Maui does.